src/test/denialofservice_tests.cpp

```diff
@@ -44,11 +46,10 @@ BOOST_FIXTURE_TEST_SUITE(denialofservice_tests, TestingSetup)
 // work.
 BOOST_AUTO_TEST_CASE(outbound_slow_chain_eviction)
 {
-    auto connman = std::make_unique<CConnman>(0x1337, 0x1337, *m_node.addrman, *m_node.netgroupman);
+    ConnmanTestMsg& connman = *static_cast<ConnmanTestMsg*>(m_node.connman.get());
```

Comment:
nit in the second commit: Can be written shorter as `static_cast<ConnmanTestMsg&>(*m_node.connman)`?

Reply:
Yes, done.